tuf/tuf.go

```diff
@@ -165,15 +184,14 @@ func (tr *Repo) RemoveBaseKeys(role string, keyIDs ...string) error {
 // will be added in front of (i.e. higher priority) in the delegation list.
 // An empty before string indicates to add the role to the end of the
 // delegation list.
-// A new, empty, targets file will be created for the new role.
 func (tr *Repo) UpdateDelegations(role *data.Role, keys []data.PublicKey, before string) error {
```

Comment:
I'm not even convinced it's a good way to do delegations, hence why I
haven't implemented it...